packages/storefront-lwc/src/modules/commerce/basket/basket.scss

```diff
@@ -1,7 +1,13 @@
+
 .line-item-attributes {
     padding-left: .3em;
 }
 
+.product-info {
```

Comment:
we should use bootstrap classes for margin and paddings unless UX is adamant on a very very specific value

Reply:
okay, good point, I am researching on how to do this. Thanks!